src/main/java/com/locallampoon/fiveh/ui/PanelStyles.java

```diff
@@ -50,6 +50,13 @@
     public static final Color PLAYER_COLOR = new Color(10,248,8);
     public static final Color NEIGHBOUR_COLOR = new Color(61,184,255);
     public static final Color FLOOR_NAME_COLOR = new Color(255,255,255);
-    public static final Font FLOOR_NAME_FONT = new Font("TimesRoman", Font.PLAIN, UNIT_SIZE*3/2);
-    public static final Font ROOM_TEXT_FONT = new Font("TimesRoman", Font.PLAIN, UNIT_SIZE*4/3);
+    public static final Font FLOOR_NAME_FONT = new Font("Monospaced", Font.PLAIN, UNIT_SIZE*3/2);
+    public static final Font ROOM_TEXT_FONT = new Font("Monospaced", Font.PLAIN, UNIT_SIZE*4/3);
```

Comment:
I used FONT_FAMILY because tried to align the design. Only my map panel using that "Monospaced" now?